lib/globalize/active_record/class_methods.rb

```diff
@@ -122,7 +122,12 @@ def define_translations_accessor(name)
       def database_connection_possible?
         begin
           # Without a connection tentative, the `connected?` function can responds with a false negative
-          ::ActiveRecord::Base.connection
+          if Globalize.rails_7_2?
+            connection = ::ActiveRecord::Base.lease_connection
+            connection.connect!
```

Comment:
I think in `Globalize::ActiveRecord::ActMacro#allow_translation_of_attributes` it should be:
```
          self.ignored_columns += translated_attribute_names.map(&:to_s)
          if database_connection_possible?
            reset_column_information
          end
```

But that made some tests red wich testet migration things e.g. `test/globalize/migration_test.rb`
I think they expect `database_connection_possible?` is false because `translation_class.table_exists?` is not set up yet.

I don't want to change to much so I went with this solution. 

Btw: `sqlite` workt because of the `load_schema` in `Globalize::Test::Database#connect` established an connection.

I think it "broke" with one of these Rails changes but I'm not sure:
- https://github.com/rails/rails/commit/2da1852ac9b1886407464b246da3b99cfa186cfc#diff-642b90553b888bd2c724c093a1a685a5408a7d8293f3751366c25dc548936eb7R246
- https://github.com/rails/rails/commit/dd8fd52c072b8f6777ad6d8acf86054fa1923dd9#diff-353f4f101b1bb04938d761b6e29c88e0f147084e9e276854a77ae5d9a9a3b2a8R10